### PR TITLE
storage/txn: add tests.

### PR DESCRIPTION
**Pull request by disksing**
@zhangjinpeng1987 @ngaut @siddontang

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: src/storage/mvcc/txn.rs [hunk 1]
```diff
@@ -282,104 +305,219 @@ mod tests {
...
+
+        must_prewrite_delete(engine.as_ref(), b"x", b"x", 15);
+        must_rollback(engine.as_ref(), b"x", 15);
+        must_seek_write(engine.as_ref(),
+                        b"x",
+                        u64::max_value(),
+                        15,
+                        15,
+                        WriteType::Rollback);
+        must_reverse_seek_write_none(engine.as_ref(), b"x", 15);
```
Comment (BusyJay) [inline src/storage/mvcc/txn.rs]: Why doesn't it seek the Put on "x" at ts 10?

Comment (disksing) [inline src/storage/mvcc/txn.rs]: Keys on CF_WRITE are in descending order, so reverse_seek_write will only return keys with greater ts.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**